### PR TITLE
Run code through isort + black

### PR DESCRIPTION
**Pull request by kylef**
This change updates the code to follow the typical Python style guide ([pep8](https://pep8.org/)) using isort and [black](https://github.com/psf/black), the Python code formatter. This results in some changes to the import order, removing some unused imports or dead code.

While it isn't enforced at the CI level, it just brings some consistency to the code base. We may desire to enforce it in the future.

### CODE REVIEW

File: polls/management/commands/cleanup.py [hunk 1]
```diff
@@ -13,16 +14,19 @@ def handle(self, *args, **kwargs):
...
-        qs = Question.objects.exclude(id__in=initial_question_pks).filter(published_at__lt=one_hour_ago)
+        qs = Question.objects.exclude(id__in=initial_question_pks).filter(
+            published_at__lt=one_hour_ago
+        )
 
         print('Deleting {} questions'.format(qs.count()))
         qs.delete()
 
-        qa = Vote.objects.all()
+        qs = Vote.objects.all()
```
Comment (kylef) [inline polls/management/commands/cleanup.py:29]: linting exposed that `qa` was an unused variable exposing the bug where deleting votes (next two lines) was a no-op and not doing the desired behaviour.

Comment (opichals) [reply polls/management/commands/cleanup.py:29]: Should we split this into a separate commit for clarity?

Comment (kylef) [reply polls/management/commands/cleanup.py:29]: Yeah you're right, I was taking shortcuts. Addressed in 122ec55

**Review by opichals — APPROVED** [review]
Thanks! :rocket:

**Pull request merged**